pandas/io/formats/style_render.py

```diff
@@ -250,8 +250,7 @@ def _translate(self, sparse_index: bool, sparse_cols: bool, blank: str = "&nbsp;
             d.update({k: map})
 
         table_attr = self.table_attributes
-        use_mathjax = get_option("display.html.use_mathjax")
-        if not use_mathjax:
+        if not get_option("styler.html.mathjax"):
```

Comment:
so this is technically a change in the option name? (its ok to merge this, just mention in the whatsnew)

Reply:
done. I will edit the whats new before 1.4.0 release.